tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+    "compilerOptions": {
+        "target": "ES6",
+        "lib": ["ES2015", "ES6"],
+        "module": "commonjs",
+        "sourceMap": true
+    }
+}
```

Comment:
Consider making your config a bit more rich.
E.g.
1. `"strict": true` - use strict mode
2. For libraries, it's usually better to have `"importHelpers": true`, so you'd share the helpers with other libraries and won't re-add them yourself
3. Emit declarations with `"declaration": true` so your consumers will have typings

Reply:
+1 for `strict: true`